guides/common/modules/con_ipv4-and-ipv6-requirements.adoc

```diff
@@ -0,0 +1,44 @@
+[id="ipv4-and-ipv6-requirements_{context}"]
+= IPv4 and IPv6 requirements
+
+You can install {Project} in an IPv4 network or in an IPv6 network.
+
+ifdef::satellite[]
+[NOTE]
+====
+Dual-stack {Project} installation that uses both IPv4 and IPv6 is not supported.
+====
+endif::[]
+
+The following requirements apply to installations in an IPv4 network:
+
+* Ensure that an IPv6 loopback is configured.
+* Do not disable IPv6 in kernel by adding the `ipv6.disable=1` kernel parameter.
++
+ifdef::satellite[]
+For a supported way to disable the IPv6 protocol, see link:https://access.redhat.com/solutions/5045841[How do I disable the IPv6 protocol on Red Hat Satellite and/or Red Hat Capsule server?] in Red{nbsp}Hat Knowledgebase.
+endif::[]
+
+The following requirements apply to installations in an IPv6 network:
+
+* You must deploy an external DHCP IPv6 server as a separate, unmanaged service to bootstrap clients into GRUB2.
+GRUB2 configures IPv6 networking by using DHCPv6 or by assigning a static IPv6 address.
+This is required because the DHCP server in {RHEL} (ISC DHCP) does not provide an integration API for managing IPv6 records, therefore the {SmartProxy} DHCP plugin that provides DHCP management is limited to IPv4 subnets.
+ifdef::satellite[]
+* You must deploy an external HTTP proxy server that supports both IPv4 and IPv6.
+This is required because Red Hat Content Delivery Network distributes content only over IPv4 networks, therefore you must use this proxy to pull content into the {Project} on your IPv6 network.
+endif::[]
+ifndef::satellite[]
+* Optional: If you rely on content from IPv4 networks, you must deploy an external IPv4 HTTP proxy server.
+This is required to access Content Delivery Networks that distribute content only over IPv4 networks, therefore you must use this proxy to pull content into {Project} on your IPv6 network.
+endif::[]
++
+You must configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy server as the default proxy.
+For more information, see {InstallingServerDocURL}adding-a-default-http-proxy_{project-context}[Adding a Default HTTP Proxy to {Project}].
+
+Although {Project} provisioning templates include IPv6 support for PXE and HTTP (iPXE) provisioning, the only tested and certified provisioning workflow is the UEFI HTTP Boot provisioning.
+This limitation only relates to users who plan to use {Project} to provision hosts.
```

Comment:
@Lennonka Is this relevant for you with regards to https://github.com/theforeman/foreman-documentation/pull/3532? I'm letting you know just in case it's something you want to look into as part of your PR. (These two lines are currently in the installation guide, my PR doesn't add them.)

Reply:
Thank you for letting me know! Yes, that will need an update. I'll take a look at it once your PR will have been merged.